core-backend/common/src/lib.rs

```diff
@@ -90,7 +90,7 @@ pub trait Environment<E: Ext + Into<ExtInfo> + 'static>: Default + Sized {
     ) -> Result<(), BackendError<'static>>;
 
     /// Run setuped instance starting at `entry_point` - wasm export function name.
-    /// NOTE: expternal environment must be setuped.
+    /// NOTE: external environment must be setuped.
```

Comment:
There is no word `setuped`.
```suggestion
    /// NOTE: external environment must be set up.
```